tvb_framework/tvb/adapters/creators/siibra_creator.py

```diff
@@ -175,8 +176,11 @@ def launch(self, view_model):
         # list of all resulting indices for connectivities and possibly connectivity measures
         results = []
 
-        conn_dict, conn_measures_dict = siibra_base.get_connectivities_from_kg(atlas, parcellation, subject_ids,
-                                                                               compute_fc)
+        try:
+            conn_dict, conn_measures_dict = siibra_base.get_connectivities_from_kg(atlas, parcellation, subject_ids,
+                                                                                    compute_fc)
+        except SiibraHttpRequestError:  # this type of Error is raised in case of invalid token, but it might change?
```

Comment:
I saw an HTTP 403 code in the actual Siibra exception in case of the invalid token.
Can you check that too ?
If that is present also (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403) then it is clear the token issue, otherwise, raise a generic message, something like:"We could not complete the operation. Please check the logs and contact the development team from TVB, siibra, EBRAINS KG"

Reply:
Thanks for the suggestion, will implement it.
I personally got only error 401, but I see in their code that 403 is also possible, so I will include both of them.